cf/tasks/package-bits.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -x
+set -o errexit
+set -o nounset
+
+#find . -print
+
+# Determine configuration.
+
+VERSION=$(cat semver.scf-version/version)
+PLATFORM=$(echo $(basename s3.certstrap-binary/certstrap-*.tgz .tgz)|sed -e 's|\(.*\)\.\([^.]*\)$|\2|')
+ARCHIVE="${PWD}/out/scf-${PLATFORM}-${VERSION}.zip"
+
+echo Packaging for $PLATFORM, taking $VERSION ...
+
+# Assembling the pieces ...
+mkdir tmp
+
+# kube configs
+# helm charts
+( cd tmp ; unzip ../s3.kube-dist/scf-kube-*.zip )
+
+# "Am I Ok" for k8s
+cp src/bin/dev/k8s-ready-state-check.sh tmp/
+
+# NOTE: Code below is a variant of `src/make/cert-generator`, modified
+# to take the certstrap binary from the task's input instead of
+# directly from AWS S3, etc. Further modified to not generate a
+# tarball, but leave things in the area of assembly.
```

Comment:
Maybe we can make `src/make/cert-generator` look for the archive at <some place defined by environment variables or something>, and only download if it's missing?

(The not making a tarball isn't as much of a deal, we can extract it after)

Mostly to avoid code drift here.